### ISSUE TITLE
Bug: AttributeError: 'Namespace' object has no attribute 'profile_name'

### ISSUE THREAD

**Issue by riskynacho** [duo-labs/cloudmapper#117]
Debian 9.4 inside a docker container. 
[Installed debian packages list](https://transfer.sh/umlvD/packages.txt)
[Installed python packages list](https://pastebin.com/1jdKnLCW)
Latest master branch

```
(venv) root@aca55ecd78e6:/cloudmapper# python cloudmapper.py collect --account nachoprod
* Getting region names
Traceback (most recent call last):
  File "cloudmapper.py", line 69, in <module>
    main()
  File "cloudmapper.py", line 64, in main
    commands[command].run(arguments)
  File "/cloudmapper/commands/collect.py", line 198, in run
    collect(args)
  File "/cloudmapper/commands/collect.py", line 99, in collect
    if arguments.profile_name:
AttributeError: 'Namespace' object has no attribute 'profile_name'
(venv) root@aca55ecd78e6:/cloudmapper# ls account-data/
demo  nachoprod
(venv) root@aca55ecd78e6:/cloudmapper# cat config.json
{
    "accounts": [
        {
            "default": true,
            "id": "nachoprod",
            "name": "nachoprod"
        }
    ],
    "cidrs": {
        "1.1.1.1/32": {
            "name": "SF Office"
        },
        "2.2.2.2/28": {
            "name": "NY Office"
        }
    }
}
(venv) root@aca55ecd78e6:/cloudmapper#
```
aws cli is working fine with the creds stored in `/root/.aws`.

Any idea what I am doing wrong? 

I was able to get the demo working without an issue. 

Side question, is there an IRC channel for this project?

Thanks in advance.

**Update:** The `collect_data.sh` script does work if that helps diagnose the problem.

**Comment by SeanLeftBelow** [comment]
Also getting the same error.
```$ python cloudmapper.py collect --account "prod" --profile "prod"
* Getting region names
Traceback (most recent call last):
  File "cloudmapper.py", line 69, in <module>
    main()
  File "cloudmapper.py", line 64, in main
    commands[command].run(arguments)
  File "/Users/sean/src/github.com/cloudmapper/commands/collect.py", line 198, in run
    collect(args)
  File "/Users/sean/src/github.com/cloudmapper/commands/collect.py", line 99, in collect
    if arguments.profile_name:
AttributeError: 'Namespace' object has no attribute 'profile_name'
```
Leaving --profile off still produces the same error. 

Looks like the last few merges broke something.
I get past the profile error on `git checkout 5fdd99bc3b56210baf846588d0973bdfe3c71298` however I still encounter a `TypeError: Unknown type` and it fails to execute fully.

**Comment by riskynacho** [comment]
Not sure if this is a related problem or a different root cause. Ater `collect_data.sh` ran, I did a `python cloudmapper.py prepare --account nachoprod` and it kicked back the following. 

```
Building data for account engprod (nachoprod)
- 0 nodes built in region ap-south-1
- 0 nodes built in region eu-west-3
- 0 nodes built in region eu-west-2
- 0 nodes built in region eu-west-1
- 0 nodes built in region ap-northeast-2
- 0 nodes built in region ap-northeast-1
- 0 nodes built in region sa-east-1
- 0 nodes built in region ca-central-1
- 0 nodes built in region ap-southeast-1
- 0 nodes built in region ap-southeast-2
- 0 nodes built in region eu-central-1
- 0 nodes built in region us-east-1
- 0 nodes built in region us-east-2
- 0 nodes built in region us-west-1
- 0 nodes built in region us-west-2
- 0 external CIDRs built
- 0 connections built
```

I can look in `account-data/nachoprod/us-east-1/describe-instances.json` and see thousands of instances (which is correct).

**Comment by 0xdabbad00** [comment]
Thank you for reporting this.  Should be fixed now in https://github.com/duo-labs/cloudmapper/pull/119

**Comment by 0xdabbad00** [comment]
Ah, and looks like I need to `update collect_data.sh` as well as it should have written the file as `ec2-describe-instances.json` not `describe-instances.json`.  I may remove that script in the future as it doesn't do nearly as much as the `collect` command now does.

**Comment by riskynacho** [comment]
I know you said it was being deprecated, I only tried it because the other method wasn't working. Thank you for the quick response =)

**Comment by 0xdabbad00** [comment]
The collect_data.sh should now be fixed in #120, but you should use the `collect` command now instead.
